legal/satzung.md

```diff
@@ -0,0 +1,123 @@
+# Satzung
+
+
+## Name, Sitz, Geschäftsjahr
+
+1.   Der Verein trägt den Namen *open energy modeling foundation* (oemof)
+2.   Sitz des Vereins ist $(SITZ).
+3.   Das Geschäftsjahr ist das Kalenderjahr.
+
+
+## Zweck und Ziele
+
+1. Der Verein verfolgt ausschließlich und unmittelbar gemeinnützige Zwecke im Sinne des Abschnitts „Steuerbegünstigte Zwecke“ der Abgabenordnung.
+2. Zweck des Vereins ist die Förderung von Wissenschaft und Forschung.
+    Dieser Zweck wird verwirklicht durch
+    1. Bereitstellung wissenschaftlicher Software unter OSI-anerkannten Open-Source-Lizenzen,
```

Comment:
Should the modelling framework "oemof" be mentioned here literally?
@joroeder : We could make clear that the aim of the oemof foundation is to support the oemof developer group and the development of packages of the oemof framework. Then, it would be more clear that oemof foundation != oemof developer group.
@p-snft : It might be difficult to explain to any of the German authorities what the "Oemof Developer Group" is and we may have to define it.